src/pkg/utils/wait.go

```diff
@@ -73,7 +75,7 @@ func ExecuteWait(waitTimeout, waitNamespace, condition, kind, identifier string,
 
 	// Setup the spinner messages.
 	conditionMsg := fmt.Sprintf("Waiting for %s%s%s to be %s.", kind, identifierMsg, namespaceMsg, condition)
-	existMsg := fmt.Sprintf("Waiting for %s%s%s to exist.", kind, identifierMsg, namespaceMsg)
+	existMsg := fmt.Sprintf("Waiting for %s%s to exist.", filepath.Join(kind, identifierMsg), namespaceMsg)
```

Comment:
on windows this will cause this message to look like: `Waiting for pod\\hello to exist.`, this needs a `filepath.ToSlash` to prevent that

Reply:
updated